partner_supplierinfo/tests/test_partner_supplierinfo.py

```diff
@@ -0,0 +1,81 @@
+# Copyright 2022 Tecnativa - Pilar Vargas
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+from odoo.tests import common
+
+
+class TestPartnertSupplierInfo(common.TransactionCase):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.partner_1 = cls.env["res.partner"].create(
+            {
+                "name": "Test partner 1",
+                "email": "partner1@test.com",
+                "is_company": True,
+            }
+        )
+        cls.partner_2 = cls.env["res.partner"].create(
+            {
+                "name": "Test partner 2",
+                "email": "partner2@test.com",
+                "is_company": True,
+            }
+        )
+        cls.partner_3 = cls.env["res.partner"].create(
+            {
+                "name": "Test partner 3",
+                "ref": "0000",
+                "email": "partner3@test.com",
+                "is_company": True,
+            }
+        )
+        cls.supplier_info_1 = cls.env["partner.supplierinfo"].create(
+            {
+                "ref": "1234",
+                "partner_id": cls.partner_1.id,
+                "supplier_id": cls.partner_2.id,
+            }
+        )
+        cls.supplier_info_2 = cls.env["partner.supplierinfo"].create(
+            {
+                "ref": "4567",
+                "partner_id": cls.partner_1.id,
+                "supplier_id": cls.partner_3.id,
+            }
+        )
+        cls.supplier_info_2 = cls.env["partner.supplierinfo"].create(
```

Comment:
You don't really need to assign them to a variable. You can simply do:

```suggestion
        cls.env["partner.supplierinfo"].create(
```

Reply:
I have deleted the variable, it 's true that it isn't used.